dev/tests/js/jasmine/tests/app/code/Magento/AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/overlay.test.js

```diff
@@ -0,0 +1,283 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+define([
+    'jquery',
+    'Magento_AdobeStockImageAdminUi/js/components/grid/column/overlay'
+], function ($, Overlay) {
+    'use strict';
+
+    describe('Magento_AdobeStockImageAdminUi/js/components/grid/column/overlay', function () {
+
+        var overlayObj;
+
+        beforeEach(function () {
+            overlayObj = new Overlay({
+                provider: 'providerName',
+                licensed: {}
+            });
+        });
+
+        describe('"initObservable" method', function () {
+
+            it('Check for defined ', function () {
+                expect(overlayObj.hasOwnProperty('initObservable')).toBeDefined();
+            });
+
+            it('Check method type', function () {
+                var type = typeof overlayObj.initObservable;
+
+                expect(type).toEqual('function');
+            });
+
+            it('licensed will be defined', function () {
+                expect(overlayObj.licensed).toBeDefined();
+            });
+        });
+
+        describe('"updateLicensed" method', function () {
+
+            var handler = {
+                success: function(data) {
+                    return [1,2]
+                },
+                fail: function(data) {
+                    return 'Error Message';
+                }
+            };
+            var testObj = {
+                ajaxFunction: function() {
+                    var promise =
+                        $.ajax({
+                            url: 'http://example.com/admin/adobe_stock/license/getlist/?ids=1,2,3',
+                            type: 'GET',
+                            data: {
+                                'form_key': 'MagentoFormkey'
+                            },
+                            dataType: 'json',
+                            context: overlayObj
+                        });
+                    promise.done(function(data, status) {
+                        handler.success(data);
+                    });
+                    promise.fail(function(status, error) {
+                        handler.fail(error);
+                    });
+                }
+            };
+
+            it('Check for defined', function () {
+                expect(overlayObj.hasOwnProperty('updateLicensed')).toBeDefined();
+            });
+
+            it('Check method type', function () {
+                var type = typeof overlayObj.updateLicensed;
+               expect(type).toEqual('function');
+            });
+
+            it("Check Ajax Success request", function() {
```

Comment:
This test needs to be reworked. This test is only exercising the fake variables/methods that are created in the housing `describe` block above, and not actually exercising the `updateLicensed` method. At a minimum, this test should:
- stub (or spy on) `$.ajax`, which you have set up properly below, to prevent an actual network request, and manipulate the ajax call to return the data you want to fake to exercise the functionality you want (in this case, you want to fake return ajax "success" data).
- invoke `Overlay.updateLicensed`